Frameworks/Foundation/NSDecimalNumber.mm

```diff
@@ -312,34 +350,98 @@ - (NSDecimalNumber*)decimalNumberByRaisingToPower:(NSUInteger)power {
  @Status Stub
  @Notes
 */
-- (NSDecimalNumber*)decimalNumberByMultiplyingByPowerOf10:(short)power {
+- (instancetype)initWithString:(NSString*)numericString locale:(id)locale {
     UNIMPLEMENTED();
     return StubReturn();
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
+*/
+- (NSDecimal)decimalValue {
+    // Read-Only
+    NSDecimal decimal = { 0 };
+    NSDecimalCopy(&decimal, &_decimal);
+    return decimal;
+}
+
+- (NSDecimalNumber*)_decimalOperation:(_DecimalOperations)operand
+                        decimalNumber:(NSDecimalNumber*)decimalNumber
```

Comment:
You could do away with the enum by using a function pointer here; it seems odd to have a four-member enum for a strictly internal method.